lib/stdlib/python/unittest/__main__.py

```diff
@@ -0,0 +1,16 @@
+"""Main entry point"""
+
+import sys
+
+# PyPy.js doesn't have an "argv" so we'll need to set it.
```

Comment:
We could, maybe as ``sys.argv = [pypyjs]``, but that may lead to more problems than it solves. Perhaps an empty list is a better value. I wonder what ``sys.argv`` is when CPython is embedded in another application... I'll try to find out.